src/mscorlib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -17,5 +18,11 @@ public static class EnvironmentAugments
         public static bool HasShutdownStarted => Environment.HasShutdownStarted;
         public static string StackTrace => Environment.StackTrace;
         public static int TickCount => Environment.TickCount;
+        public static string GetEnvironmentVariable(string variable) => Environment.GetEnvironmentVariableCore(variable);
+        public static string GetEnvironmentVariable(string variable, EnvironmentVariableTarget target) => Environment.GetEnvironmentVariableCore(variable, target);
```

Comment:
Just because there is custom logic here- I didn't want to duplicate it.  It is simple enough though- I'll yank it then we don't have to surface the enum.